src/networking.c

```diff
@@ -2871,6 +2884,50 @@ int clientSetNameOrReply(client *c, robj *name) {
     return result;
 }
 
+/* Check validity of an attribute that's gonna be shown in CLIENT LIST. */
+int validateClientAttrOrReply(client *c, sds attr, sds val) {
```

Comment:
the problem is that validateClientName returns a static string as `char*` output arg.
so i can't use sprintf to mention the exact argument that's violating (if i wanna return a static string).
so i opted to create an `OrReply` method (which isn't suitable for the callers of validateClientName.

while in my new method, i wanted it to mention the name of the bad attr, and be easy to use (since i called it in multiple places, for multiple arguments, in one command).

maybe now, that i use it in my new command just once, mentioning the bad attribute name is not critical.
however for SETNAME it's still a good idea since it's also used in HELLO.

so the only way i can maybe unify them, it to create a function that just validates the content and returns a boolean, without any string reply.
i'll do that.